docs/src/pages/[platform]/connected-components/liveness/react-props.ts

```diff
@@ -78,13 +78,13 @@ export const FACE_LIVENESS_DETECTOR_CONFIG = [
   {
     name: `binaryPath?`,
     description:
-      'Overrides the WASM backend binary CDN path, the default is https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-wasm@3.11.0/dist/. When overriding this path ensure that the wasm version matches the version of @tensorflow/tfjs-backend-wasm installed by npm.',
+      'Overrides the WASM binary path, the default is https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow/tfjs-backend-wasm/3.11.0/. When overriding this path ensure that the wasm version matches the version of @tensorflow/tfjs-backend-wasm installed by npm.',
```

Comment:
Same question here to make it easier to copy:

```suggestion
      'Overrides the WASM binary path, the default is `https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow/tfjs-backend-wasm/3.11.0/`. When overriding this path ensure that the wasm version matches the version of @tensorflow/tfjs-backend-wasm installed by npm.',
```